src/plays/tenzies-game/styles.css

```diff
@@ -0,0 +1,59 @@
+.main {
```

Comment:
Please use a prefix style name(unique to your play) so that we do not have any conflict chances with other plays.

Reply:
Got it, will change it.